iosApp/Survey/Sources/Presentation/Views/FadePaginationView.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  FadePaginationView.swift
+//  Survey
+//
+//  Created by Bliss on 28/9/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct FadePaginationView<T>: View {
+
+    @Binding private var currentPage: Int
+    @State private var currentVisible: Double = 1.0
```

Comment:
`currentVisible` sounds a bit weird as there is not a noun presenting the variable. Please correct me if I am wrong but it should be `currentVisibleThreshold` or `currentVisibility`, right?

Reply:
Updated in https://github.com/blyscuit/kmm-swiftUI-IC/pull/57/commits/de04d90f5ac9e5bbc3b76ee7f3127b8ae887468e